libse/SubtitleFormats/UnknownSubtitle41.cs

```diff
@@ -115,15 +116,15 @@ private static string EncodeTimeCode(TimeCode time)
         {
             int frames = MillisecondsToFrames(time.TotalMilliseconds);
             int footage = frames / 16;
-            int rest = (int)Math.Round(frames % 16.0 / 16.0 * Configuration.Settings.General.CurrentFrameRate);
+            int rest = (int)Math.Round(frames % 16.0 / 16.0 * DefaultFrameRate);
             return string.Format("{0}.{1:0}", footage, rest);

```

Comment:
TODO: this line.
string.Format("{0}.{1:0}", footage, rest); means different thing... **rest** is a int type, so formatting has no effect here!

If you try: `$"{10:0}"` will still get 10.